### ISSUE TITLE
node_modules is ignored in package > exclude statement in serverless.yml

### ISSUE THREAD

**Issue by horike37** [serverless/serverless#3938]
# This is a (Bug Report)

## Description
For bug reports:
* What went wrong?
It didn't exclude from a deployment package if you describe `node_modules` in `package > exclude` statement in serverless.yml. Is this a bug or by design?

**Step to reproduce**
1. Describe `node_modules` in `package > exclude` statement in serverless.yml
```yaml
package:
  exclude:
    - node_modules/**
```
2. Run `sls package`

* What did you expect should have happened?

I expected the package was created with excluding `node_modules`. However the package has included  `node_modules`. 

If you specify a directory name other than `node_modules`, it is excluded correctly.

* What was the config you used?
Here is my serverless.yml

```yml
service: sls-test-2

provider:
  name: aws
  runtime: nodejs6.10

package:
  exclude:
    - package-lock.json
    - node_modules/**
    - test/**

functions:
  hello:
    handler: handler.hello
```
* What stacktrace or error message from your provider did you see?
nothing

## Additional Data

* ***Serverless Framework Version you're using***: 1.17.0
* ***Operating System***: macOS Sierra 10.12.5
* ***Stack Trace***: nothing
* ***Provider Error messages***: nothing

**Comment by horike37** [comment]
Here is my project tree.
```
├── handler.js
├── node_modules
├── package-lock.json
├── package.json
├── serverless.yml
└── test
    └── testfile
```
Then attached the zip package generated by `sls package`
[sls-test-2.zip](https://github.com/serverless/serverless/files/1148175/sls-test-2.zip)

**Comment by HyperBrain** [comment]
Doesn't excluding the `node_modules` conflict with the dependency packaging of Serverless? I mean the `node_modules` folder in the generated package is not a result of folder exclusion but a result of dependency packaging.
@pmuens Correct me if I'm wrong.

**Comment by pmuens** [comment]
Thanks for opening @horike37 and thanks for commenting @HyperBrain! :+1:

It seems like this bug is caused by https://github.com/serverless/serverless/pull/3889 and possibly a related to / a duplicate of https://github.com/serverless/serverless/issues/3869.

**Comment by horike37** [comment]
This issue has been resolved on v1.18.1
just closed
